perllib/Integrations/SCP.pm

```diff
@@ -119,13 +119,15 @@ sub pay {
                             'scpbase:amountInMinorUnits' => $args->{amount},
                             'scpbase:reference' => $self->config->{customer_ref},
                         ),
-                        'scpbase:tax' => {
-                            'scpbase:vat' =>ixhash(
-                                'scpbase:vatCode' => $self->config->{scp_vat_code},
-                                'scpbase:vatRate' => $self->config->{scp_vat_rate} || 0,
-                                'scpbase:vatAmountInMinorUnits' => $args->{vat} || 0,
-                            ),
-                        },
+                        $args->{vat} ? (
+                            'scpbase:tax' => {
+                                'scpbase:vat' =>ixhash(
+                                    'scpbase:vatCode' => $self->config->{scp_vat_code},
+                                    'scpbase:vatRate' => $self->config->{scp_vat_rate} || 0,
+                                    'scpbase:vatAmountInMinorUnits' => $args->{vat} || 0,
+                                ),
+                            },
+                        ) : (),
```

Comment:
Might be worth checking if this should be included even if it's zero rated in case they need it for accounting purposes. Mostly I say this as they did include a vat code in the tech spec.

Reply:
Ah, that's the Bromley figures, they haven't updated that part at all. But yes, will check.